src/coreclr/tools/aot/jitinterface/CMakeLists.txt

```diff
@@ -11,8 +11,14 @@ if(CLR_CMAKE_TARGET_WIN32)
   set(JITINTERFACE_RESOURCES Native.rc)
 endif()
 
+if(CLR_CMAKE_TARGET_BROWSER)
```

Comment:
This one is an interface between crossgen2 and jit, so it is a shared library built for the platform hosting the build tools, not for the target.